front/lib/api/assistant/pubsub.ts

```diff
@@ -145,6 +147,13 @@ async function handleUserMessageEvents(
                 );
                 break;
               }
+              case "agent_generation_cancelled":
```

Comment:
should be in the list above that post to message channel no?

Reply:
Because I'm returning and not breaking to end the loop, and same below!

Reply:
actually removed cause I cannot return if multiple messages are generating 🙏🏻